src/Microsoft.OpenApi/Extensions/StringExtensions.cs

```diff
@@ -16,22 +17,21 @@ public static class StringExtensions
         /// Gets the enum value based on the given enum type and display name.
         /// </summary>
         /// <param name="displayName">The display name.</param>
-        public static T GetEnumFromDisplayName<T>(this string displayName)
+        [UnconditionalSuppressMessage("Trimming", "IL2075", Justification = "Fields are never trimmed for enum types.")]
```

Comment:
```suggestion
```

This shouldn't be needed, right? Since you have a `DynamicallyAccessedMembers` attribute on the `T`.

Reply:
Yep -- we can remove this.